kornia/filters/kernels.py

```diff
@@ -24,6 +24,21 @@ def gaussian(window_size: int, sigma: float) -> torch.Tensor:
     return gauss / gauss.sum()
 
 
+def n_gaussian(window_size: int, sigma: torch.Tensor) -> torch.Tensor:
```

Comment:
Can we just update the previous function accepting both float and tensor ? If float then convert to tensor. No need code duplication

Reply:
I updated the code to do that